Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -218,15 +218,39 @@ WarpX::ComputeFaceExtensions(){
             + std::to_string(N_ext_faces_after_eight_ways(2))
         );
     }
+
+    bool using_bck = false;
+
+    // If any cell could not be extended we use the BCK method to stabilize them
+#if !defined(WARPX_DIM_XZ) && !defined(WARPX_DIM_RZ)
     if (N_ext_faces_after_eight_ways(0) > 0) {
-        amrex::Abort(Utils::TextMsg::Err("Some x faces could not be extended"));
+        ApplyBCKCorrection(0);
+        using_bck = true;
     }
+#endif
+
     if (N_ext_faces_after_eight_ways(1) > 0) {
-        amrex::Abort(Utils::TextMsg::Err("Some y faces could not be extended"));
+        ApplyBCKCorrection(1);
+        using_bck = true;
     }
+
+#if !defined(WARPX_DIM_XZ) && !defined(WARPX_DIM_RZ)
     if (N_ext_faces_after_eight_ways(2) > 0) {
-        amrex::Abort(Utils::TextMsg::Err("Some z faces could not be extended"));
+        ApplyBCKCorrection(2);
+        using_bck = true;
     }
+#endif
+
+    if(WarpX::verbose and using_bck) {
+        amrex::Print() << Utils::TextMsg::Info(
```

Comment:
Could you use the function [WarpX::RecordWarning](https://github.com/ECP-WarpX/WarpX/blob/development/Source/WarpX.cpp#L436) instead for this?

Reply:
In this case, you could also probably remove the `if` condition on `WarpX::verbose`.

Reply:
Unfortunately the warning makes the CI fail. Is there a way to fix this?

Reply:
Yes, you can add `runtime_params = warpx.abort_on_warning_threshold=medium` to the entry corresponding to the test `embedded_boundary_rotated_cube` in Regression/WarpX-tests.ini. This should fix the issue.

Reply:
Amazing! It works 😄